### PR TITLE
github: introduce settings app

### LINKED ISSUE #374: Add settings app
Any objections to adding the https://github.com/apps/settings app to the org?

It introduces a small attack window where users with push access can change the settings of their repo. But it also makes the config more transparent and self-serving. IMO the trade-off is worth it.

### PR DESCRIPTION
**Pull request by zimbatm**
self-configure using the settings app.

Fixes #374

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  enable_automated_security_fixes: true
+
+  # Either `true` to enable vulnerability alerts, or `false` to disable
+  # vulnerability alerts.
+  enable_vulnerability_alerts: true
+
+# Labels: define labels for Issues and Pull Requests
+#
+labels:
+# NOTE: leave that up to the https://github.com/numtide/.github repo
```
Comment (zowoq) [inline .github/settings.yml:61]:
```suggestion
```

**Review by zowoq — COMMENTED** [review]
Looks fine, one nit.

**Comment by zimbatm** [comment]
bors merge

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  name: infra
+
+  # A short description of the repository that will show up on GitHub
+  description: nix-community infrastructure [maintainer=@Mic92]
+
+  # A URL with more information about the repository
+  homepage: "https://nix-community.org"
+
+  # A comma-separated list of topics to set on the repository
+  topics: [terraform, nixos]
```
Comment (zowoq) [inline .github/settings.yml:14]: Is the app working, these haven't been added to the repo?

Comment (zimbatm) [reply .github/settings.yml:14]: Fixed in https://github.com/nix-community/infra/commit/433613dd0eb96849086bf34722fa1ed10ef0a1fb . Turns out it's stringly typed.

**Pull request merged**